tools/scylla-sstable.cc

```diff
@@ -929,6 +930,7 @@ void validate_operation(schema_ptr schema, reader_permit permit, const std::vect
         writer.EndObject();
     }
     writer.EndStream();
+    fmt::print(std::cout, "{}", ss.str());
```

Comment:
```suggestion
    std::cout << ss.view();
```

for two reasons:

1. simpler this way. as {fmt} does not help with the performance, readability or maintainability here.
2. less mem copy. as `ss.str()` creates a shiny new `std::string` instance which is dropped on the floor away after its inserted into `cout`. yeah, i admit this is not that important in this use case, though.

Reply:
I would prefer to use `fmt::print()` consistently in the entire `scylla-sstable.cc` file.